packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx

```diff
@@ -724,7 +724,7 @@ export class VerticalStackedBarChartBase extends React.Component<
         });
         const rectFocusProps = !shouldFocusWholeStack && {
           'data-is-focusable': !this.props.hideTooltip,
-          'aria-labelledby': `toolTip${this._calloutId}`,
+          'aria-label': this._getAriaLabel(singleChartData, point),
```

Comment:
What is the issue with callout content. Is that not always upto date

Reply:
In browse mode, when the screen reader's cursor is moved to bars using Ctrl + arrow keys, it doesn't trigger focus event and the callout stays hidden with outdated content

![image](https://user-images.githubusercontent.com/110246001/203298636-1187ece9-e972-44a5-a448-44ee14fc8f64.png)